File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PatientCheckInsUpdatePatientCheckInsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PatientCheckInsUpdatePatientCheckInsRequestItem(
    @JsonProperty("CheckInDate")
    val checkInDate: String? = null,
    @JsonProperty("CheckInLocationId")
    val checkInLocationId: Int? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("PatientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("RegistrationExpiryDate")
    val registrationExpiryDate: String? = null,
    @JsonProperty("RegistrationStartDate")
    val registrationStartDate: String? = null
)

File: sdks/thunkmetrc-go/wrapper/models/patient_check_ins_update_patient_check_ins_request_item.go
package models

type PatientCheckInsUpdatePatientCheckInsRequestItem struct {
    CheckInDate string `json:"CheckInDate,omitempty"`
    CheckInLocationId int `json:"CheckInLocationId,omitempty"`
    Id int `json:"Id,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    RegistrationExpiryDate string `json:"RegistrationExpiryDate,omitempty"`
    RegistrationStartDate string `json:"RegistrationStartDate,omitempty"`
}
